markets/dagstore/miner_api.go

```diff
@@ -0,0 +1,184 @@
+package dagstore
```

Comment:
The reason why it's here it's because it's specific to Lotus and we have been working under the assumption that go-fil-markets is **not** Lotus specific.